### PR TITLE
tidb: add doc for create index

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? 

add related session/global variable in create index docs.

### Which version does your change affect? 

dev, v3.0, v2.1

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by winkyao** [comment]
@lilin90 PTAL

File: dev/reference/sql/statements/create-index.md [hunk 1]
```diff
@@ -81,6 +81,10 @@ mysql> CREATE UNIQUE INDEX c1 ON t1 (c1);
 Query OK, 0 rows affected (0.31 sec)
 ```
 
+## 相关 session 变量
+
+和 `CREATE INDEX` 语句相关的 global 变量有 `tidb_ddl_reorg_worker_cnt`, `tidb_ddl_reorg_batch_size`, `tidb_ddl_reorg_priority`, 具体可以参考 [TiDB 特定系统变量](/reference/configuration/tidb-server/tidb-specific-variables.md/#tidb-ddl-reorg-worker-cnt)
```
Comment (lilin90) [inline dev/reference/sql/statements/create-index.md]:
```suggestion
和 `CREATE INDEX` 语句相关的全局变量有 `tidb_ddl_reorg_worker_cnt`，`tidb_ddl_reorg_batch_size` 和 `tidb_ddl_reorg_priority`，具体可以参考 [TiDB 特定系统变量](/reference/configuration/tidb-server/tidb-specific-variables.md#tidb-ddl-reorg-worker-cnt)。
```

File: v2.1/reference/sql/statements/create-index.md [hunk 2]
```diff
@@ -81,6 +81,10 @@ mysql> CREATE UNIQUE INDEX c1 ON t1 (c1);
 Query OK, 0 rows affected (0.31 sec)
 ```
 
+## 相关 session 变量
+
+和 `CREATE INDEX` 语句相关的 global 变量有 `tidb_ddl_reorg_worker_cnt`, `tidb_ddl_reorg_batch_size`, `tidb_ddl_reorg_priority`, 具体可以参考 [TiDB 特定系统变量](/reference/configuration/tidb-server/tidb-specific-variables.md/#tidb-ddl-reorg-worker-cnt)
```
Comment (lilin90) [inline v2.1/reference/sql/statements/create-index.md]:
```suggestion
和 `CREATE INDEX` 语句相关的全局变量有 `tidb_ddl_reorg_worker_cnt`，`tidb_ddl_reorg_batch_size` 和 `tidb_ddl_reorg_priority`，具体可以参考 [TiDB 特定系统变量](/reference/configuration/tidb-server/tidb-specific-variables.md#tidb-ddl-reorg-worker-cnt)。
```

File: v3.0/reference/sql/statements/create-index.md [hunk 3]
```diff
@@ -81,6 +81,10 @@ mysql> CREATE UNIQUE INDEX c1 ON t1 (c1);
 Query OK, 0 rows affected (0.31 sec)
 ```
 
+## 相关 session 变量
+
+和 `CREATE INDEX` 语句相关的 global 变量有 `tidb_ddl_reorg_worker_cnt`, `tidb_ddl_reorg_batch_size`, `tidb_ddl_reorg_priority`, 具体可以参考 [TiDB 特定系统变量](/reference/configuration/tidb-server/tidb-specific-variables.md/#tidb-ddl-reorg-worker-cnt)
```
Comment (lilin90) [inline v3.0/reference/sql/statements/create-index.md]:
```suggestion
和 `CREATE INDEX` 语句相关的全局变量有 `tidb_ddl_reorg_worker_cnt`，`tidb_ddl_reorg_batch_size` 和 `tidb_ddl_reorg_priority`，具体可以参考 [TiDB 特定系统变量](/reference/configuration/tidb-server/tidb-specific-variables.md#tidb-ddl-reorg-worker-cnt)。
```

**Comment by lilin90** [comment]
@crazycs520 Please address the comments.

**Pull request merged**